branding/constants/filter.js

```diff
@@ -0,0 +1,2 @@
+export const SHOW_CONTENT_FILTER_HEADER_MENU = false
```

Comment:
why is this not in the config?

Reply:
We have a lot of things like this not in the config, but in constants as `SHOW_GROUP_BUTTON_IN_HEADER` for example.

May we have to clean this up systematicly later …